2017-02-08  Anmol Jain  <classaja@lnxsrv07.seas.ucla.edu>
	* timespec.h: modified timespec_cmp to fix bug and provide desired output
--- coreutils-with-bug/lib/timespec.h	2005-09-21 23:05:39.000000000 -0700
+++ /Assignment\ 4/coreutils-with-bug/lib/timespec.h	2017-02-08 16:30:23.598410000 -0800
@@ -45,8 +45,9 @@
 static inline int
 timespec_cmp (struct timespec a, struct timespec b)
 {
-  int diff = a.tv_sec - b.tv_sec;
-  return diff ? diff : a.tv_nsec - b.tv_nsec;
+  if(a.tv_sec > b.tv_sec) return 1;
+  else if(a.tv_sec < b.tv_sec) return -1;
+  else return 0;
 }